src/main/java/com/example/controller/SimulationMenuController.java

```diff
@@ -54,4 +69,20 @@ public void setEnabled(boolean enabled) {
     public void setChaneStateToSimulationEnabled(boolean enabled) {
         simulateItem.setDisable(!enabled);
     }
+
+    public void bindItems() {
+        SimulationController simulationController = appController.getSimulationController();
+        ObjectProperty<ApplicationState> applicationState = appController.getApplicationStateProperty();
+
+        startItem.disableProperty().bind(simulationController.getStartProperty()
+                .or(applicationState.isEqualTo(ApplicationState.DRAWING)));
```

Comment:
instead of .or() we could define setEnabled() method, as in DrawMenuController. This method can be called when we change application state to DRAWING.